less/common/App.less

```diff
@@ -27,13 +27,16 @@
       .box-shadow(0 2px 6px @shadow-color);
     }
   }
-  .App-primaryControl, .App-titleControl, .App-backControl {
+  .App-primaryControl,
```

Comment:
Can we revert the style changes? On a related note, we should look into a formatter for less.

Reply:
Whoops! Fixed that. Also, yes we should :P